Dockerfile.builder

```diff
@@ -37,5 +37,6 @@ RUN apt-get update && \
 # Install Go
 #
 RUN curl -sfL https://dl.google.com/go/go${GO_VERSION}.linux-${ARCH}.tar.gz | tar -xzC /usr/local && \
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0 \
```

Comment:
I don't think this is necessary is it?

Reply:
I feel it will be required due to changes in Makefile target, now precheck will call lint target, which requires golangci-lint. Let me know if I am wrong, will revert accordingly.

On this note, I might need to make changes in packer-ci-build to install golangci-lint binary. Will do it once this PR is in good shape.

Reply:
@aanm please let me know if you are fine with above or have any other comment. Thanks :+1: